test/models/poll.test.js

```diff
@@ -32,21 +31,8 @@ describe('Polls', () => {
   });
 
   describe('correctly validate', () => {
-    it('if poll has no title', (done) => {
```

Comment:
Maybe I'm a nerd but I love seeing tests needing to be updated :heart_eyes: Means they were working!

Reply:
As long as you are content with the tests are being updated and don't care that all I'm doing is deleting lines from the test and not adding any :P

Reply:
Meh, at some point you gotta prioritise work speed over TDD
I might go back and update them some day :wink: